# This message is for storing bounding boxes in a generic format
float32 x1
float32 y1
float32 x2
float32 y2
float32 score
int16 class_id